glib/src/translate.rs

```diff
@@ -682,6 +731,11 @@ impl<'a> ToGlibPtr<'a, *mut c_char> for OsString {
         let tmp = os_str_to_c(self);
         Stash(tmp.as_ptr() as *mut c_char, tmp)
     }
+
+    #[inline]
+    fn to_glib_full(&self) -> *mut c_char {
+        OsStr::to_glib_full(&*self)
+    }
 }
```

Comment:
Ideally all the changes that add more FFI conversions for these types should be a separate commit